### PR TITLE
Add fundraiser links

### PR DESCRIPTION
**Pull request by cbeauchesne**
Add fundraiser links based on hardcoded criterions.

### CODE REVIEW

File: src/js/get-fundraiser.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+export default function(document) {
+  // Only areas, waypoints, routes
+  if (!['r', 'w', 'a'].includes(document.type)) {
+    return null;
+  }
+
+  if (document.type === 'r') {
+    // for routes, only mountain_climbing and rock_climbing activity
+    if (intersection(document.activities, ['mountain_climbing', 'rock_climbing']).length === 0) {
```
Comment (brunobesson) [inline src/js/get-fundraiser.js]: `if (!intersection(document.activities, ['mountain_climbing', 'rock_climbing']).length)` is enough :)

File: src/js/get-fundraiser.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  // for waypoints, only climbing outdoor sites
+  if (document.type === 'w' && document.waypoint_type !== 'climbing_outdoor') {
+    return null;
+  }
+
+  const areas = [];
+  if (document.type === 'a') {
+    areas.push(document);
+  } else {
+    areas.push(...document.areas.filter((area) => area.area_type === 'range'));
```
Comment (brunobesson) [inline src/js/get-fundraiser.js]: `areas.push(...document.areas.filter((area) => ['range', 'admin_limits', 'country'].includes(area.area_type));`

File: src/js/get-fundraiser.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    areas.push(...document.areas.filter((area) => area.area_type === 'admin_limits'));
+    areas.push(...document.areas.filter((area) => area.area_type === 'country'));
+  }
+
+  for (const area of areas) {
```
Comment (brunobesson) [inline src/js/get-fundraiser.js]:
```js
return areas.map(area => mapping[area.document_id]).find((m) => m !== null)
```

could be enough?

File: src/js/get-fundraiser.js (hunk 2 above)
Comment (cbeauchesne) [reply src/js/get-fundraiser.js]: Nope, the point of these three lines is to have first ranges, then admins_limit, then countries.

The logic behind is to display the item attached to the smallest entity. I also plan to include waypoints, and even routes in that logic : 

```
If an item is attached to the route => display it
else if an item is attached to it's main waypoint  => display it
else if an item is attached to an area of type range => display it
else if an item is attached to an area of type admin_limit => display it
else if an item is attached to an area of type country => display it
```

File: src/js/get-fundraiser.js (hunk 3 above)
Comment (cbeauchesne) [reply src/js/get-fundraiser.js]: Yes, but IMO less readable.

File: src/js/get-fundraiser.js (hunk 1 above)
Comment (cbeauchesne) [reply src/js/get-fundraiser.js]: 👍

File: src/js/get-fundraiser.js (hunk 3 above)
Comment (brunobesson) [reply src/js/get-fundraiser.js]: up to you

**Pull request merged**